rules.d/meson.build

```diff
@@ -4,31 +4,30 @@ install_data(
         'README',
         install_dir : udevrulesdir)
 
-rules = files('''
-        60-autosuspend.rules
-        60-block.rules
```

Comment:
This reformatting breaks the Fedora-only BFQ patch:

https://src.fedoraproject.org/rpms/systemd/blob/rawhide/f/use-bfq-scheduler.patch

which, by extension, breaks Packit, so I guess the patch needs to be regenerated in the Rawhide branch.

Reply:
I'll do it after this is merged. It doesn't make much sense to do it before.